content/mysql/concepts/build-in-functions/terms/covarPop/covarPop.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'MySQL: COVAR_POP() function' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Covariance built-in function comparing the population of two cells in a table.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Built-In Function'
+  - 'MySQL'
```

Comment:
```suggestion
Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
  - 'Built-In Function'
  - 'MySQL'
```
Include only those subjects that are part of the subjects.md list, if you feel that the list is insufficient then you can add a new subject in the [subjects.md file.](documentation/subjects.md)
Also, remove the comment ' # Please only use Subjects in the subjects.md file, for  more readability.